examples/python/tests/drivers/test_options.py

```diff
@@ -29,3 +31,69 @@ def test_page_load_strategy_none():
 
     driver.get("http://www.google.com")
     driver.quit()
+
```

Comment:
Anyone know how I can test these files locally? Local env points to upstream trunk.

Reply:
hi @pmartinez1 ,

Here are the instructions
https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/examples/python/README.md

Reply:
HI @pmartinez1 ,

Here are the instructions to run tests locally, https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/examples/python/README.md

Reply:
Hi @harsha509 ,

I had followed those steps, but the links to the examples in my local server were pointing to the /trunk branch, so my newly added code-block examples were not showing up. I realized I needed to update the code-blocks to point to my local branch for those to be pulled in. Looks to be all there now. Ready for review.

Screen recording. 

https://github.com/user-attachments/assets/903c3952-d19b-4d1f-be4b-d27b7f7888a3

Reply:
Ah,

I usually commit code samples in one commit and then link to them in another commit. Occasionally, I carefully select line numbers from local files to include in the docs. This approach has consistently worked for me! :smile:

To answer your question, there's no other method available right now!